rockcraft/lifecycle.py

```diff
@@ -36,23 +37,46 @@ def pack():
     except OSError as err:
         raise CraftError(err) from err
 
+    name = yaml_data.get("name")
+    if not name:
+        emit.message("Project name not specified.")
```

Comment:
aren't these errors?

Reply:
Yes. Will change the emit method accordingly.